uiowa/uiowa
# Emergency endpoint cleanup

Resolves #7778 





# How to test


Sync a site, any site
_If_ there happens to be an active hawk alert, see it!

Assuming not, then set the emergency endpoint to the stage endpoint, and see that the test hawk alert(s) appear

example with default site:
```
ddev drush @default.local cset uiowa_alerts.settings hawk_alert.source https://emergency.stage.drupal.uiowa.edu/api/active && ddev drush @default.local cr
```
